images/azure/rocky-91.yaml

```diff
@@ -6,7 +6,7 @@ packer:
   distribution_version: "rockylinux-9"  # SKU
   # Azure Rocky linux official image: https://portal.azure.com/#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/erockyenterprisesoftwarefoundationinc1653071250513.rockylinux-9
   image_publisher: "erockyenterprisesoftwarefoundationinc1653071250513"
-  image_version: "latest"
+  image_version: "9.1.0"
```

Comment:
I think this might not work. we will need `9.1.20230215` from the output given in the PR description.

Reply:
yep